src/app/api/notification/notification.service.ts

```diff
@@ -0,0 +1,104 @@
+import { CopilotAPI } from '@/utils/CopilotAPI'
+import { BaseService } from '@api/core/services/base.service'
+import { AssigneeType, Task } from '@prisma/client'
+import { NotificationTaskActions } from '../core/types/tasks'
+import { z } from 'zod'
+import { CompanyResponse, CopilotUser, MeResponse } from '@/types/common'
+import { getEmailDetails, getInProductNotificationDetails } from './notification.helpers'
```

Comment:
import fix

Reply:
Fixed in 3a9dcb86db254fe7b133359ecb68d8b0b1d765be commit